src/lib/analytics/apis.native.js

```diff
@@ -29,10 +29,16 @@ class GoogleWrapper {
     assign(this, { analytics })
   }
 
-  async setDefaultParams(params = {}) {
+  async identify(userId) {
     const { analytics } = this
 
-    await analytics.setDefaultEventParameters(params)
+    await analytics.setUserId(userId)
```

Comment:
don't we need to update package `@react-native-firebase/analytics` or it's already supported in the current version ?

Reply:
yes it is supported